go/test/endtoend/fix_replication_test.go

```diff
@@ -0,0 +1,296 @@
+// Copyright 2025 Supabase, Inc.
```

Comment:
Ideally this test can be moved to endtoend/multiorch with some shared setup for all multiorc tests. @deepthi  I will for your PR for that. But this is a basic test that we can port there once your PR related to this lands